mmv1/templates/terraform/resource.html.markdown.tmpl

```diff
@@ -191,12 +191,15 @@ This resource does not support import.
 * `{{replaceAll $idFormat "%" "" }}`
 {{- end }}
 
-
-In Terraform v1.5.0 and later, use an [`import` block](https://developer.hashicorp.com/terraform/language/import) to import {{$.Name}} using one of the formats above. For example:
+In Terraform v1.12.0 and later, use an [`identity` block](https://developer.hashicorp.com/terraform/language/resources/identities) to import {{$.Name}} using identity values. For example:
```

Comment:
should we still link the import block together with the identities block?

Reply:
Perhaps remove the `import` block in favor of `identity` block instead? I'm leaning towards having the two options still present to give users the decision on what they'd like to use for importing resources.